policyengine_us/variables/gov/states/oh/tax/income/deductions/medical_exepenses/oh_insured_unreimbursed_medical_care_expenses.py

```diff
@@ -0,0 +1,56 @@
+from policyengine_us.model_api import *
+
+
+class oh_insured_unreimbursed_medical_care_expenses(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Ohio Unreimbursed Medical and Health Care Expense Deduction"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://tax.ohio.gov/static/forms/ohio_individual/individual/2022/it1040-sd100-instruction-booklet.pdf#page=18",  # Line 36
+        "https://tax.ohio.gov/static/forms/ohio_individual/individual/2022/it1040-sd100-instruction-booklet.pdf#page=27",
+        "https://codes.ohio.gov/ohio-revised-code/section-5747.01",  # R.C. 5747.01(10)
+    )
+    defined_for = StateCode.OH
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        employer_premium_contribution = person(
+            "employer_contribution_to_health_insurance_premiums_category",
+            period,
+        )
+        status = employer_premium_contribution.possible_values
+        medicare_eligible = person("is_medicare_eligible", period)
+        # Line 3
+        eligible_premiums = (
+            person("health_insurance_premiums", period) * medicare_eligible
```

Comment:
don't need the select statement -- they're not eligible unless this condition is met
```suggestion
            person("health_insurance_premiums", period) * medicare_eligible * (employer_premium_contribution == status.NONE)
```